README.md

```diff
@@ -0,0 +1,20 @@
+# Photino!
+
+[![Travis](https://api.travis-ci.org/victorporof/tofino.svg?branch=bfg-browser)](https://travis-ci.org/victorporof/tofino)
+
+An implementation of Photon styling on top of Tofino.
+
+## How to run
+1. `git clone git@github.com:victorporof/tofino.git`
+2. `cd tofino`
+3. `git checkout bfg-browser`
+4. `npm install`
+
+* To develop:
+  1. `npm start` is for the optimized build.
+  2. `npm run dev+logging` is for the slow/Chrome debugger build.
+  3. `npm run serve+logging -- --browser firefox --mock-os linux` for running in Firefox with our nice devtools.
```

Comment:
I'd detail a bit more that `--mock-os` can also be `win32` or `darwin`. Leaving the `--browser` option out will just open in your default browser.